src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -251,6 +251,67 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify refresh-incremental rebuild of index.") {
+    Seq(("csv", Map("header" -> "true")), ("parquet", Map()), ("json", Map())).foreach {
+      case (format, option: Map[String, String]) =>
+        spark.conf.set(IndexConstants.REFRESH_APPEND_ENABLED, true)
```

Comment:
naive question: didn't we decide in an earlier PR to use `withSQLConf`?

Reply:
Thanks @rapoth ,